src/plugins/intel_gpu/src/kernel_selector/kernels/convolution/convolution_kernel_base.h

```diff
@@ -36,10 +36,8 @@ class ConvolutionKernelBase : public WeightBiasKernelBase {
             size_t globalWorkSizeDZ;
         };
 
-        union {
-            CLDNNStyle cldnnStyle;
-            GEMMStyle gemmStyle;
-        };
+        CLDNNStyle cldnnStyle;
+        GEMMStyle gemmStyle;
```

Comment:
could you explain why you removed union here?

Reply:
Union share the memory for two data type CLDNNStyle and GEMMstyle. But the value are set and initialized on mixed way. Need to be separate data memory.

Reply:
If this variable is used in a mixed way, I think that is an issue. But I think introducing `GetJitConstantsWithLoopUnroll` function would resolve the mixed use of this variable. isn't it? The `union` here is an explicit statement that the conv runs either in cldnn style or gemm style. So mixed use looks very weird.

Reply:
For example, convolution_kernel_bfyx_os_iyx_osv16 kernel doesn't use gemmStyle. But the gemmStyle has some value during set cldnnStyle values because they are Union. And loopCount should be possible maximum number for all cases and it uses gemmStyle values got from garbage value of cldnnStyle.